app-backend/batch/src/main/scala/com/azavea/rf/batch/export/spark/Export.scala

```diff
@@ -64,7 +64,7 @@ object Export extends SparkJob with Config with LazyLogging {
           val targetRDD: TileLayerRDD[SpatialKey] =
             ed.output.rasterSize match {
               case Some(size) => rdd.regrid(size)
-              case None => rdd
+              case None => 4000
```

Comment:
it needs to actually do the regrid operation here and reproject / regrid on lines 132-140 in every case